Studies/source/MvaTraining.cxx

```diff
@@ -39,13 +42,15 @@ struct Arguments { // list of all program arguments
     REQ_ARG(std::string, range);
     OPT_ARG(size_t, number_sets, 2);
     OPT_ARG(uint_fast32_t, seed, std::numeric_limits<uint_fast32_t>::max());
+    OPT_ARG(uint_fast32_t, seed2, 123456);
```

Comment:
Just to be sure, please, ask what seed and number of splits Maria Teresa is using.